src/main/resources/assets/react/documentTypes/RemoveFieldFromDocumentTypeModal.jsx

```diff
@@ -77,45 +79,75 @@ export function RemoveFieldFromDocumentTypeModal({
 		size='small'
 	>
 		<Modal.Header>{global
-			? `Remove override of global field ${name}`
-			: `Delete local field ${name}?`}
+			? `Remove ${name} from ${documentTypeName}`
+			: `Delete ${name}?`}
 		</Modal.Header>
 		<Modal.Content>
 			{global ? null : <>
 				<p>If there are any graphql clients out there, which use this local field, deleting it will cause the very next query to throw an error!</p>
 				<p>Deactivating a field is safe, and a better option, unless you are certain the field is not in use...</p>
 			</>}
-			<p>This documentType is used by the following...</p>
+			{fieldHasValueInDocumentsTotal > 0 ?
```

Comment:
Even though a field is not present in an actual document, it may still be used in an interface GraphQL query. Thus it's still important to warn about breakage if field is removed...

Reply:
@poi33 I guess we need something like

`const isFieldInUse = fieldHasValueInDocumentsTotal + interfacesArr.length collectionsArr.length > 0;`

and then check `isFieldInUse` before displaying the entire block.